kore/src/Kore/OnePath/Verification.hs

```diff
@@ -235,5 +237,104 @@ verifyClaim
         StrategyPattern.Stuck p : _ -> throwE p
         StrategyPattern.Bottom : _ -> error "Unexpected bottom pattern."
   where
-    liftIO' :: IO () -> ExceptT (CommonExpandedPattern level) Simplifier ()
-    liftIO' = liftIO
+    transitionRule'
+        :: Prim (CommonExpandedPattern level) (RewriteRule level Variable)
+        -> (StrategyPattern (CommonExpandedPattern level), StepProof level Variable)
+        -> Simplifier [(StrategyPattern (CommonExpandedPattern level), StepProof level Variable)]
+    transitionRule' =
+        OnePath.transitionRule
+            metadataTools
+            substitutionSimplifier
+            simplifier
+            axiomIdToSimplifier
+
+-- | TODO: Docs.
+type Configuration level = StrategyPattern (CommonExpandedPattern level)
```

Comment:
Could you add documentation to this type synonym? I think you could also use it above.

Reply:
This is intended to be internal in the module and only as an alias to avoid ridiculously long types; not sure I have any docs to write to it.

Reply:
Okay, I ended up defining this right below `StrategyPattern` as `CommonStrategyPattern` and replacing all occurrences.